var/spack/repos/builtin/packages/bowtie2/package.py

```diff
@@ -34,6 +35,7 @@ class Bowtie2(Package):
     # seems to have trouble with 6's -std=gnu++14
     conflicts('%gcc@6:', when='@:2.3.1')
     conflicts('@:2.3.5.0', when='target=aarch64:')
+    conflicts('@2.4.1', when='target=aarch64:')
```

Comment:
```suggestion
    conflicts('@:2.4.1', when='target=aarch64:')
```

Reply:
I wanted to target only 2.4.1 for conflicts.
I didn't want to target the buildable version 2.3.5.1.